### PR TITLE
LP-1595 Add dynamic page for my badges in courseware

### PR DESCRIPTION
**Pull request by hassanarif426**
### Description

[LP-1595](https://philanthropyu.atlassian.net/browse/LP-1595)

[philu-edx-theme PR](https://github.com/philanthropy-u/philu-edx-theme/pull/1067)

Add a description of your changes here.

### CODE REVIEW

File: openedx/features/badging/views.py [hunk 1]
```diff
@@ -38,9 +38,11 @@ def trophycase(request):
 @require_GET
 @login_required
 def my_badges(request, course_id):
+    """ this function returns badges related to on course """
     user = request.user
 
     course_key = CourseKey.from_string(unicode(course_id))
+    course = get_course_with_access(user, "load", course_key)
```
Comment (zeshanarifios) [inline openedx/features/badging/views.py]: quotes inconsistent, Please use single quotes

File: lms/djangoapps/commerce/migrations/0001_data__add_ecommerce_service_user.py [hunk 2]
```diff
@@ -19,9 +19,6 @@ def backwards(apps, schema_editor):
     User.objects.get(username=USERNAME, email=EMAIL).delete()
 
 class Migration(migrations.Migration):
-    dependencies = [
-        ('onboarding', '0003_historicaluser'),
-    ]
```
Comment (zeshanarifios) [inline lms/djangoapps/commerce/migrations/0001_data__add_ecommerce_service_user.py]: Are we removing this intentionally?

File: common/djangoapps/student/migrations/0009_auto_20170111_0422.py [hunk 3]
```diff
@@ -7,7 +7,7 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('student', '0017_auto_20171219_0925'),
+        ('student', '0008_auto_20161117_1209'),
```
Comment (zeshanarifios) [inline common/djangoapps/student/migrations/0009_auto_20170111_0422.py]: Are we updating this intentionally?

File: common/djangoapps/student/migrations/0016_historicaluserprofile.py [hunk 4]
```diff
@@ -11,7 +11,7 @@ class Migration(migrations.Migration):
 
     dependencies = [
         migrations.swappable_dependency(settings.AUTH_USER_MODEL),
-        ('student', '0008_auto_20161117_1209'),
+        ('student', '0019_auto_20181221_0540'),
```
Comment (zeshanarifios) [inline common/djangoapps/student/migrations/0016_historicaluserprofile.py]: Are we updating this intentionally?

File: openedx/features/badging/tabs.py [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+class BadgingTab(CourseTab):
+    """ The my badges new tab view. """
+    type = 'my_badges'
+    title = ugettext_noop('My Badges')
+    priority = 20
+    view_name = 'my_badges'
+    tab_id = 'my_badges'
+    is_dynamic = True
+    is_default = False
+
```
Comment (zeshanarifios) [inline openedx/features/badging/tabs.py]: How about adding `is_hideable = True` to make it hideable

Comment (emraan004) [reply openedx/features/badging/tabs.py]: No need to make it hide-able at the moment.

**Comment by saad-waqar** [comment]
LGTM 👍

**Pull request merged**